hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/server/HermesServer.java

```diff
@@ -151,4 +159,23 @@ private HttpHandler handlers() {
     private boolean isFrontendRequestDumperEnabled() {
         return hermesServerParameters.isRequestDumperEnabled();
     }
+
+    @VisibleForTesting
```

Comment:
If I understand correctly, this annotation makes the method visible in tests. So do we need this annotation when the method is public, or do we need the public access modifier?

Reply:
> If I understand correctly, this annotation makes the method visible in tests. 

Nope, this annotation is from Guava and doesn't change anything, it only states that the method was created only for tests. But your question is still valid whether we need it or not. When comes to me, it's ok to use it as it states explicite that the method was created only for test purposes and maybe you shouldn't use it in production.